--- /usr/share/X11/xkb/rules/evdev.lst	2019-02-11 22:25:45.000000000 +0300
+++ xkb/rules/evdev.lst	2022-01-27 09:35:53.753825184 +0300
@@ -629,6 +629,8 @@
   phonetic_azerty ru: Russian (phonetic, AZERTY)
   phonetic_dvorak ru: Russian (phonetic, Dvorak)
   phonetic_fr     ru: Russian (phonetic, French)
+  ruchey_cyrillic ru: Russian (Engineering, Cyrillic)
+  ruchey_latin    ru: Russian (Engineering, Latin)
   yz              rs: Serbian (Cyrillic, ZE and ZHE swapped)
   latin           rs: Serbian (Latin)
   latinunicode    rs: Serbian (Latin, Unicode)
